pwiz_tools/Skyline/TestPerf/SmallMolDriftTimePredictorTest.cs

```diff
@@ -104,6 +104,10 @@ protected override void DoTest()
                 editIonMobilityLibraryDlg.CreateDatabaseFile(databasePath); // Simulate user click on Create button
                 editIonMobilityLibraryDlg.GetIonMobilitiesFromResults();
             });
+
+            // Make sure we haven't broken this dialog's use of TargetResolver (if we have, we'll probably get the serializable format)
+            AssertEx.AreEqual(@"Sulfamethizole", editIonMobilityLibraryDlg.GetTargetDisplayName(0));
```

Comment:
I would recommend putting this Assert inside of the RunUI right above.
I usually end up with infrequent intermittent failures if I write code which interacts with a DataGridView from the test thread.

Reply:
Good tip, thanks.